app/views/uploads/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_tag({:action => "upload_zip"}, :multipart => true) do %>
+
+  <div class="row mb-3 input-group">
+    <%= label :library_pick, "Library:", class: "col-sm-2 col-form-label" %>
+    <div class='col-sm-10 ps-0'>
+      <%= select "post", "library_pick", Library.all.map{ |lib| [lib.name, lib.id] }, {include_blank: false}, {class: "form-control form-select"} %>
+      <span id="libraryPick" class="form-text">The library to upload to.</span>
+    </div>
+  </div>
+
+  <div class="row mb-3 input-group">
+    <%= label :upload, 'Select File', for: 'upload', class: "col-sm-2 col-form-label" %>
+    <div class='col-sm-10 ps-0'>
+      <%= file_field 'upload', 'datafiles', {multiple: true} %>
```

Comment:
I think we should restrict to .zip files in the file dialog, if that's all that's supported at the moment.

Reply:
```suggestion
      <%= file_field 'upload', 'datafiles', {multiple: true, accept: 'application/zip'} %>
```